protocol/communities/manager_archive.go

```diff
@@ -354,6 +354,10 @@ func (m *ArchiveManager) StartHistoryArchiveTasksInterval(community *Community,
 				m.logger.Error("failed to get community chat topics ", zap.Error(err))
 				continue
 			}
+			// adding the content-topic used for member updates.
+			// since member updates would not be too frequent i.e only addition/deletion would add a new message,
+			// this shouldn't cause too much increase in size of archive generated.
+			topics = append(topics, m.transport.FilterByChatID(community.MemberUpdateChannelID()).ContentTopic)
```

Comment:
But why do we need this information in the archive? 🤔  Just the final community description is not enough?

Reply:
From now on all chat messasges will be published on this topic, we want them to land in the archive.

Reply:
oh indeed, makes sense.
Though the comment doesn't make sense then I guess

Reply:
will update the comment to include this info. Also wanted to indicate that along with chats memberUpdates would also get archived which is a side-effect. 
Hopefully it won't take up too much of archive space though.